src/docs/deployment/android.md

```diff
@@ -514,11 +544,13 @@ The resulting app bundle or APK files are located in
 [GitHub repository]: {{site.github}}/google/bundletool/releases/latest
 [Google Maven]: https://maven.google.com/web/index.html#com.google.android.material:material
 [gradlebuild]: {{site.android-dev}}/studio/build/#module-level
-[Issue 9253]: {{site.repo.flutter}}/issues/9253
-[Issue 18494]: {{site.repo.flutter}}/issues/18494
+[Issue 9253]: {{site.github}}/flutter/flutter/issues/9253
```

Comment:
Should we use the `repo.flutter` definition? Any consideration for changing this?